packages/loader/container-definitions/src/deltas.ts

```diff
@@ -104,6 +104,9 @@ export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>
     /** The last sequence number processed by the delta manager */
     readonly lastSequenceNumber: number;
 
+    /** The last message processed by the delta manager */
+    readonly lastMessage: ISequencedDocumentMessage | undefined;
```

Comment:
@arinwt i was also thinking this could be useful for some of the stuff done in the summarizer, and might allow reducing some of the op tracking. i know i wanted something like this when computing summary timeout from last op